README.md

```diff
@@ -11,7 +11,7 @@ This is an AWS IoT Thing simulator for nRF91. It shows how to use the Device API
 ### Most basic usage
 The most basic usage is just creating a device. For that you just need an API key. The device ID will be randomly generated and the rest of the necessary information (mqtt endpoint, cert, and mqtt prefix) will be pulled from the device API. 
 
-The host defaults to the production environment (https://api.nrfcloud.com). **Nordic Semiconductor personnel**: if you want to use the simulator on our `dev` or `beta` environments, set a `STAGE` env var to `dev` or `beta` (you will need an account for that environment).
+The host defaults to the production environment (https://api.nrfcloud.com). **Nordic Semiconductor personnel**: if you want to use the simulator on our `dev` or `beta` environments, set a `API_HOST` env var to `https://api.dev.nrfcloud.com` or `https://api.beta.nrfcloud.com` (you will need an account for that environment). You can also set the `API_HOST` env var to the url of your sub account (ie `https://api.<user_id>.nrfcloud.com`) and the `stage` will automatically set to `dev` (the stage is used for the job subscription topics for FOTA).
```

Comment:
```suggestion
The host defaults to the production environment (https://api.nrfcloud.com). **Nordic Semiconductor personnel**: if you want to use the simulator on our `dev`, `feature`, or `beta` environments, set a `API_HOST` env var to `https://api.[dev|feature|beta].nrfcloud.com` (you will need an AWS account for that environment). You can also set the `API_HOST` env var to the url of your sub account (ie `https://api.<user_id>.nrfcloud.com`) and the `stage` will be automatically set to `dev` (the stage is used for the job subscription topics for FOTA). You no longer need to set a `STAGE` variable, and it will be ignored.
```

Reply:
Nice clarifications. Thanks!